examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -521,7 +535,9 @@ class Recipe {
   final String author;
   final String description;
   final String imagePath;
+  final String imagePackage;
   final String ingredientsImagePath;
+  final String ingredientsImagePackage;
```

Comment:
[nit] I guess we could use the same package for both `image` and `ingredientsImage`.
Your call.

Reply:
I'll leave it as in theory the packages could be different.